.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libuavcan"]
+	path = libuavcan
+	url = git@github.com:UAVCAN/libuavcan.git
```

Comment:
I don't think adding the library as a submodule to its driver repo is a sensible thing to do. That creates a yet another reverse dependency to maintain in sync with library releases. Also, I envision that the driver repo will frequently be used as a submodule itself alongside the library core submodule; that would create redundancy with the current approach as libuavcan would be submoduled twice.

I suggest cloning the latest release tag (or just master) of libuavcan to run tests instead.

Reply:
Okay. I'll add a git clone to the makefile script.